.github/workflows/ci-plugin-server.yml

```diff
@@ -102,6 +102,12 @@ jobs:
             - name: Check builds correctly
               run: pnpm --filter=@posthog/plugin-server build
 
+            - name: Inject sourcemaps
+              run: pnpm --filter=@posthog/plugin-server sourcemap:inject
+
+            - name: Upload sourcemaps
+              run: pnpm --filter=@posthog/plugin-server sourcemap:upload
```

Comment:
Saw the other comment about this from @oliverb123 (not in a comment thread booooo ;) ) 

I don't exactly know what happens when this happens 😅 Like is this meant to be for the deployed version or is it okay to be uploading just "whenever" - here it will upload whenever we run the CI checks which i assume is not what we want?